src/Mantasflowers.Contracts/ServiceAgents/Common/Converters/BooleanJsonConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace Mantasflowers.Contracts.ServiceAgents.Common.Converters
+{
+    public class BooleanJsonConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(bool);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            return reader.Value.ToString().ToLower().Trim() switch
+            {
+                "true" or "yes" or "y" or "1" => true,
+                "false" or "no" or "n" or "0" => false,
+                _ => new JsonSerializer().Deserialize(reader, objectType),
+            };
+        }
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+        }
```

Comment:
What is this? Can we delete it?

Reply:
It is required to implement inherited abstract methods. The purpose of converter is to level out or rather make parcel monkey's api consistent, since instead of a boolean, they return "Y", which is pretty bad.